commands/cluster-slot-stats.md

```diff
@@ -0,0 +1,95 @@
+`CLUSTER SLOT-STATS` returns an array of slot usage statistics for slots assigned to the current shard.
+The command is suitable for Valkey Cluster users aiming to assess general slot usage trends, identify hot / cold slots, migrate slots for a balanced cluster workload, and / or re-write application logic to better utilize slots.
+
+The following statistics are supported:
+* `key-count` - Number of keys residing in a given slot.
+* `cpu-usec` - Amount of cpu time (in micro-seconds) spent on a given slot.
+* `network-bytes-in` - Amount of network ingress (in bytes) received for given slot.
+* `network-bytes-out` - Amount of network egress (in bytes) sent out for given slot.
+
+## Supported filtering and ordering arguments
+There exist two mutually exclusive arguments for controlling the output, namely;
+
+### SLOTSRANGE
+Returns slot statistics based on the slots range provided. The range is inclusive.
+The `SLOTSRANGE` argument allows for request pagination.
+The response is ordered in ascending slot number.
```

Comment:
The syntax is also inserted automatically above this text, from the JSON file. We should document the options. If they're mutually exclusive, the syntax will display this as `< SLOTRANGE xxxx | ORDERBY yyy >` so I don't think we need to mention that.

```suggestion
## Options

* `SLOTSRANGE`: Returns slot statistics based on the slots range provided.
  The range is inclusive.
  The `SLOTSRANGE` argument allows for request pagination.
  The response is ordered in ascending slot number.
```